### PR TITLE
Added My_Enumerable Assignment

### PR DESCRIPTION
**Pull request by storrence88**

### CODE REVIEW

File: steven_torrence/Fundamentals/my_enumerable.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+module MyEnumerable
+    def my_each
+        i = 0
+        while i < self.length
```
Comment (mthwtkr) [inline steven_torrence/Fundamentals/my_enumerable.rb:4]: Nice job doing this without using the built in .each method!

File: steven_torrence/Ruby_OOP/lion.rb [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        self
+    end
+
+    def eat_humans
+        @health += 20
+        p "The lion just ate a human."
+        self
+    end
+
+    def display_health
```
Comment (mthwtkr) [inline steven_torrence/Ruby_OOP/lion.rb:26]: I believe this method is just overriding the parent display_health method. I wonder if there is a way to call this method and have it use the parent display_health and also the lion's display_health method. hint: super

File: steven_torrence/Ruby_OOP/ruby_mathdojo.rb [hunk 3]
```diff
@@ -0,0 +1,27 @@
+class MathDojo
+    def initialize
+        @start = 0
+    end
+    def add *params
+        if params.class == Array
+            @start += params.flatten.inject(0) {|sum, i|  sum + i }
```
Comment (mthwtkr) [inline steven_torrence/Ruby_OOP/ruby_mathdojo.rb:7]: Good use of flatten

File: steven_torrence/Ruby_OOP/lion.rb (hunk 2 above)
Comment (storrence88) [reply steven_torrence/Ruby_OOP/lion.rb:26]: Okay, I was confused about that. If I use super, I still need to have a method in the lion class that  prints the updated text, right?

**Pull request merged**